index.html

```diff
@@ -67,5 +67,130 @@ <h5 class="small-text">I can help you build a product , feature or website Look
+            <li class="tag-item">Bootstrap</li>
+            <li class="tag-item">Ruby</li>
+           </ul>
+           <button type="button">See Project</button>
+        </div>
+        <div class="works-grid">
+           <div class="work-item">
+            <div class="work-item-content">
+                <h3>Multi-Post Stories</h3>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="tags">
+                    <li class="tag-item">CSS</li>
+                    <li class="tag-item">HTML</li>
+                    <li class="tag-item">Bootstrap</li>
+                    <li class="tag-item">Ruby</li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+           </div> 
+           <div class="work-item">
+            <div class="work-item-content">
+                <h3>Multi-Post Stories</h3>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="tags">
+                    <li class="tag-item">CSS</li>
+                    <li class="tag-item">HTML</li>
+                    <li class="tag-item">Bootstrap</li>
+                    <li class="tag-item">Ruby</li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+        </div>
+        <div class="work-item">
+            <div class="work-item-content">
+                <h3>Multi-Post Stories</h3>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="tags">
+                    <li class="tag-item">CSS</li>
+                    <li class="tag-item">HTML</li>
+                    <li class="tag-item">Bootstrap</li>
+                    <li class="tag-item">Ruby</li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+            </div>
+            </div>
+    </section>
+
+    <section class="about-page">
+      <h1 class="large-intro">
+      About Me  
+      </h1>  
+      <p class="about-para">
+        Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
```

Comment:
- [ ] Kindly use the correct font sixes for this section as provided by the Figma design.

**This is how the section looks like:**
![image](https://user-images.githubusercontent.com/86318284/185756936-354f568d-fc28-4c27-ae1b-944c03c47993.png)
**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/185756946-df245524-7c7b-4091-b67e-9492ec05c0d1.png)